### ISSUE TITLE
Image description for 413 seems incorrect

### ISSUE THREAD

**Issue by semeano** [httpcats/http.cat#79]
Hey! 👋 

The code description for 413 is `Payload Too Large` ([RFC](https://tools.ietf.org/html/rfc7231#section-6.5.11)).
On the [homepage](https://http.cat) is correct, but the [picture](https://http.cat/413) says `Request Entity Too Large`.

Cheers! :v:

**Comment by Ryuno-Ki** [comment]
That was the [old value](https://tools.ietf.org/html/rfc2616#section-10.4.14).

**Comment by semeano** [comment]
Yeah I was suspecting of that since I've found some references to that old description.
